### DISCUSSION TITLE
Nice Work

### DISCUSSION THREAD

**Discussion by insidesmart** [benallfree/pocketpages#5] [Q&A]
Just want to congratulate you on the work done so far.

Just curious. since you have taken some inspiration from sveltekit, do you have plans to support the following features ?

- Rest Parameters in Routing - For unknown route segments ( https://kit.svelte.dev/docs/advanced-routing#rest-parameters)
- 404 pages ( https://kit.svelte.dev/docs/advanced-routing#rest-parameters )
- View Transitions Support (https://svelte.dev/blog/view-transitions)
- Image Optimizations ( https://sveltekit.io/blog/sveltekit-images#the-sveltekit-enhanced-img-package )

Kindly apologize if it is already done and I have missed it.

Looking forward

**Comment by benallfree** [comment]
I’d like to see it grow in this direction. Would you be interested in contributing?

**Comment by insidesmart** [comment]
Thanks for being open to collaborate. I am definitely keen to try pocketpages for some of my projects. The features I have listed above are the items which I might miss if I move from sveltekit. Not sure if I am competent enough to contribute the features. Neverthless will check it out if i can contribute in some way.
